libs/cart/src/reducers/cart-order-state.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { DaffCartOrderResult } from '../models/cart-order-result';
+
+export interface DaffCartOrderReducerState<T extends DaffCartOrderResult = DaffCartOrderResult> {
+  order: T;
```

Comment:
I think this should be called something else to be more clear that it isn't actually an order. Maybe just cartOrderResult?

Reply:
changed